datalad/core/local/create.py

```diff
@@ -459,7 +459,8 @@ def __call__(
 
         if _seed is None:
             # just the standard way
-            uuid_id = uuid.uuid1().urn.split(':')[-1]
+            # use a fully random identifier (i.e. UUID version 4)
+            uuid_id = uuid.uuid4().urn.split(':')[-1]
```

Comment:
Just curious, how is this different from `str(uuid.uuid4())`?

Reply:
The outcome looks identical. But I was struck by the sophistication of the code and felt attached. Now that you made me look:

```
3b8431e304 datalad/distribution/dataset.py (Michael Hanke      2016-09-01 15:38:47 +0200 154)                 self._id = uuid.uuid1().urn.split(':')[-1]
```

That was already in Datalad 0.3 ;-)

Pushed a change. thx!